src/components/lists/TaskTypeList.vue

```diff
@@ -130,6 +148,47 @@ export default {
 
     onBodyScroll (event, position) {
       this.$refs.headerWrapper.style.left = `-${position.scrollLeft}px`
+    },
+
+    updatePriorityAssets () {
+      this.assetsItems.forEach((item, index) => {
+        index += 1
+        const form = {
+          id: item.id,
+          name: item.name,
+          priority: String(index),
+          allow_timelog: String(item.allow_timelog || 'true')
+        }
+        item.priority = index
+        this.$emit('update', form)
+      })
+    },
+
+    updatePriorityShots () {
+      this.shotsItems.forEach((item, index) => {
+        index += 1
+        const form = {
+          id: item.id,
+          name: item.name,
+          priority: String(index),
+          for_shots: String(item.for_shots || 'true'),
```

Comment:
For some reason that I don't totally get, if I don't specify `for_shots` & `allow_timelog` here the existing values change from `true` to `false`…